bundles/ch.elexis.core.findings.util/src/ch/elexis/core/findings/util/fhir/transformer/SlotTerminTransformer.java

```diff
@@ -50,8 +56,16 @@ public Optional<IAppointment> getLocalObject(Slot fhirObject) {
 
 	@Override
 	public Optional<IAppointment> updateLocalObject(Slot fhirObject, IAppointment localObject) {
-		// TODO lock
+		historyService = AppointmentHistoryServiceHolder.get();
+		String originalArea = localObject.getSchedule();
+		LocalDateTime oldStartTime = localObject.getStartTime();
+		LocalDateTime newStartTime = fhirObject.getStart().toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime()
+				.withSecond(0).withNano(0);
```

Comment:
Weshalb wird diese Start Zeiten geholt, aber nicht verglichen und entsprechend ein history Eintrag erstellt?

Reply:
Da die Aktivierung erst erfolgt, wenn das Objekt in einen anderen Bereich verschoben wird, ist ein Vergleich mit der Zeit hier nicht erforderlich. Der History-Eintrag wird korrekt erstellt und enthält sowohl die alte als auch die neue Zeit. Die Startzeit wird zu diesem Zeitpunkt abgerufen, da` localObject.getStartTime()` nach Ausführung von `attributeMapper.fhirToElexis(fhirObject, localObject);` bereits mit der neuen Zeit aktualisiert ist.